src/lib/zf/log.c

```diff
@@ -143,6 +144,8 @@ int zf_log_redirect(const char* file)
 
   zf_log_stderr(); /* close any existing file */ 
   zf_log_file = f;
+  strcpy(zf_log_file_name, file);
```

Comment:
Would you be able to change any `strcpy` calls to `strncpy`? Although I do wonder in this case if we should warn the user (or even error) on truncation.

Reply:
I've changed it to strncpy and issued a warning if truncated. Thanks!